docs/Key-Manager-Support.md

```diff
@@ -9,6 +9,52 @@ When using an external CA, the system must be configured with:
 
 Once configured, the Key Manager will generate and manage keys tied to the uploaded certificates and use them to sign Verifiable Credentials.
 
+## VC Signing with External CA-Signed Certificates
+Inji Certify supports the use of externally issued, CA-signed certificates for credential signing, enabling issuers to integrate their own `Public Key Infrastructure (PKI)` into the credential issuance workflow.
+
+### Trust Element: Public Key Infrastructure (PKI)
+PKI is the backbone of digital trust in credential issuance. It uses cryptographic keys and certificates to verify the authenticity and integrity of digital credentials.
```

Comment:
the word backbone sounds we are supporting PKI trust comparing others like did:web etc.. which is not true. Lets say PKI is one of the important ways to get digital trust.